cumulus/ansible/tasks/providers/ec2.py

```diff
@@ -187,6 +187,15 @@ def get_volume(self, girder_volume):
 
         return volume
 
+    def get_machine_images(self, name, owner):
+        filters = [{
+            'Name': 'name',
+            'Values': [name]
+        }]
+        r = self.client.describe_images(Owners=[owner], Filters=filters)
+
+        return [{'image_id': image['ImageId']} for image in r['Images']]
```

Comment:
@cjh1 It looks like Owner can be added to the filters argument passed to to [describe_images](http://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.Client.describe_images).  How do you feel about changing the signature of this function to  ```def get_machine_images(self, filters=None)``` 

That way other cloud providers can implement their own logic for filters and (in the future)  we can refactor this functionality into an internal function and build a provider specific mapping (e.g.  ```provider.get_image('ubuntu', '16.04')```)